### ISSUE TITLE
Ensure cuda SDK is on all build machines

### ISSUE THREAD

**Issue by sxa555** [AdoptOpenJDK/openjdk-infrastructure#851]
From @keithc-ca:
`Only the x64 Linux build is CUDA-enabled. I suspect the build machines for Windows and ppc64le Linux don’t have the CUDA toolkit installed in the expected place.`

EDIT: Requirement from the OpenJ9 call today is to have this ready for the next quarterly release

**Comment by sxa555** [comment]
Autodetection added in https://github.com/AdoptOpenJDK/openjdk-build/pull/1187/files

**Comment by sxa555** [comment]
Should be on all build machines now - will keep an eye out and see if any miss it. Separate PR to enable it on CentOS/ppc64le but I've run it on there already so it's ready to go and should build with it tonight. This issue should close once the PR is approved/mereged

**Comment by sxa555** [comment]
@keithc-ca That path does exist on the windows machine that the referenced job was on, although it just contains 3 `.txt` files - I'm guessing from what you've said that's not right?

**Comment by sxa555** [comment]
OK the playbook for that isn't installing it fully then. I'll put it on the machines today and assign to @willsparker to fix the playbook.

**Comment by sxa555** [comment]
Removing `CUDAToolkit_9.0` from https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/8558f94fecce59121d07f59164c686184d7cb351/ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml#L19 appears to make it install everything, including the files we want but I suspect there's a smaller set we could do with selecting in the silent installer.

**Comment by keithc-ca** [comment]
According to [1], I think the command you want is:
```
C:\temp\cuda_9.0.176_windows_network-exe.exe -s compiler_9.0
```

[1] https://docs.nvidia.com/cuda/archive/9.0/cuda-installation-guide-microsoft-windows/index.html

**Comment by keithc-ca** [comment]
Verified that CUDA is enabled in `OpenJDK8U-jdk_x64_windows_openj9_2019-08-31-14-43.zip` and `OpenJDK11U-jdk_x64_windows_openj9_2019-09-03-05-14.zip`. Thanks, everyone.
